Makefile

```diff
@@ -15,17 +15,17 @@ REPO_ROOT := $(shell git rev-parse --show-toplevel)
 BIN_DIR := bin
 OUT_DIR := out
 FLINTLOCKD_CMD := cmd/flintlockd
-TOOLS_DIR := hack/tools
-TOOLS_BIN_DIR := $(TOOLS_DIR)/bin
-TOOLS_SHARE_DIR := $(TOOLS_DIR)/share
+BUILD_TOOLS_DIR := hack/build
```

Comment:
I'm also not sure about the rename of the current `hack/tools` as its a common pattern within the Kubernetes community for using specific versions of external tools. 

It feels like our own tools/frameworks/etc that aid our testing should sit under `test/`.

Reply:
gah didn;t realise it was a k8s convention (although to me it feels like a mistake for them to have put it there, as i would not expect tools like that to live under a dir called `hack` at all but o well 🤷‍♀️ )

i will move it out

Reply:
although i would like to point out that the main reason i am doing this is for the supreme satisfaction of doing it all with rebase and having it go perfectly with no errors 😏